src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializer
+from sentry.models import (
+    Group, GroupCommitResolution, Project,
+    Release, ReleaseCommit,
+)
+
+
+class IssuesResolvedInReleaseEndpoint(OrganizationEndpoint):
+
+    def get(self, request, organization, project_slug, version):
+
+        release = Release.objects.get(version=version)
+        project = Project.objects.get(slug=project_slug)
```

Comment:
same comment as a above, this should be `project = Project.objects.get(slug=project_slug, organization=organization)` to avoid `MultipleObjectsReturned`